pkg/routes/quickstarts.go

```diff
@@ -17,14 +17,15 @@ func FindQuickstartById(id int) (models.Quickstart, error) {
 	return quickStart, err
 }
 
-func findQuickstartsByName(name string, pagination Pagination) ([]models.Quickstart, error) {
+func findQuickstartsByDisplayName(displayName string, pagination Pagination) ([]models.Quickstart, error) {
 	var quickStarts []models.Quickstart
-	err := database.DB.Limit(pagination.Limit).Offset(pagination.Offset).Where("name = ?", name).Find(&quickStarts).Error
-	if err != nil {
-		return nil, err
-	}
+	err := database.DB.
+		Limit(pagination.Limit).
+		Offset(pagination.Offset).
+		Where("content->'spec'->>'displayName' ILIKE ?", "%"+displayName+"%").
```

Comment:
I am wondering if we want some in-memory cache at some point (not now). The data does not change between deployments so it might be worth a show if we discover there are some performance drawbacks.

Reply:
We will see what the performance will be like in practice. If it will have a negative impact on the performance, then we will definitely explore options for optimization